Puml

betasecond opened the pull request:
Fix issue #6 
- 增加对plantuml的支持

betasecond committed:
plantuml support

betasecond committed:
plantuml support

betasecond committed:
Add PlantUML support for 复制且输出puml 复制且输出md或txt或puml

betasecond committed:
Add PlantUML support for 复制且输出puml 复制且输出md或txt或puml
- 修改了选项排序

betasecond committed:
Update 生成plantuml_子项.reg

Add PlantUML support for 复制且输出puml 
复制且输出md或txt或puml 
- 修改了选项排序

betasecond committed:
Merge remote-tracking branch 'origin/puml' into puml

betasecond committed:
添加对 PlantUML 的支持并更新文档

H1DDENADM1N commented:
我遇到些问题，运行tree2puml.exe生成文件内容仅

‘’‘
@startuml
scale 103500 width
scale 2200 height
@enduml
’‘’

betasecond commented:
> 我遇到些问题，运行tree2puml.exe生成文件内容仅
> 
> ‘’‘ @startuml scale 103500 width scale 2200 height @enduml ’‘’

`tree2puml.exe` 主要实现了目录树（tree）的功能。因此，在没有任何子目录或文件的目录下运行它，会导致输出只有 PlantUML 的基本框架。
关于输出内容：
- `@startuml` 和 `@enduml` 是 PlantUML 的开始和结束标记。
- `scale 103500 width` 和 `scale 2200 height` 是用来指定 PlantUML 图表的尺寸，因为在线网站默认尺寸不适合大项目的完整文件结构。
-

betasecond committed:
支持只有一个子目录的文件夹路径合并

- 优化 generate_plantuml_content 函数，以识别和合并只包含单一子目录的路径。
- 此更改特别支持 Java 等语言的编程规范，其中空的包结构是常见的。
- 现在，如果一个文件夹只包含一个子目录并且没有其他文件，路径将被合并，简化了生成的 PlantUML 结构表示。
- 增加了必要的路径验证，以防止将文件错误地视为目录。
